### PR TITLE
Remove ugorji JSON codecs

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by sttts** [comment]
[test]

File: hack/verify-generated-json-codecs.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+echo "===== Verifying Generated Ugorji JSON Codecs ====="
+
+if output=`find ${OS_ROOT}/vendor/k8s.io/kubernetes -name "types.generated.go" | grep -v "client-go"`
```
Comment (stevekuznetsov) [inline hack/verify-generated-json-codecs.sh]: Prefer:

    $(expr)

Over deprecated

    `expr`

Comment (stevekuznetsov) [inline hack/verify-generated-json-codecs.sh]: Also this check won't work:

```
$ find * -name somethingbad
$ echo $?
0
```

File: hack/verify-generated-json-codecs.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+then
+  echo "FAILURE: Verification of existing ugorji JSON codecs failed. These should NOT exist:"
```
Comment (stevekuznetsov) [inline hack/verify-generated-json-codecs.sh]: We have helpers now:

```
os::log::fatal "Verification of existing ugorji JSON codecs failed. These should NOT exist:\n${output}"
```

File: hack/verify-generated-json-codecs.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  echo "$output"
+  exit 1
+fi
+
+# ex: ts=2 sw=2 et filetype=sh
```
Comment (stevekuznetsov) [inline hack/verify-generated-json-codecs.sh]: Not in origin?

**Comment by liggitt** [comment]
lgtm once @stevekuznetsov's bash issues are addressed

**Comment by liggitt** [comment]
superceded by https://github.com/openshift/origin/issues/14680

File: hack/verify-generated-json-codecs.sh (hunk 1 above)
Comment (liggitt) [inline hack/verify-generated-json-codecs.sh]: I'm not sure why this was excluding client-go... we don't want generated types there either

Comment (sttts) [inline hack/verify-generated-json-codecs.sh]: The client-go codecs had been preserved in a previous carry commit, for whatever reason.

Comment (sttts) [inline hack/verify-generated-json-codecs.sh]: `find | grep` does.

**Pull request closed without merging**